test/harden/child_process.js

```diff
@@ -307,6 +307,20 @@ for (const name of functions) {
     assertProcess(t, cp.spawn(command, [], { env: { custom: 'custom' } }), { stdout: 'custom' });
   });
 
+  test('spawn(command, options) - prevent object prototype pollution', (t) => {
+    const pathName = path.join(__dirname, '_node_script.js');
+    const options = {};
+
+    // eslint-disable-next-line no-proto
+    options.__proto__['2'] = {
+      env: {
+        NODE_OPTIONS: `--require ${pathName}`,
+      },
+      shell: process.argv[0],
+    };
+    assertProcess(t, cp.spawn(command, []));
```

Comment:
Are options doing anything?  Shouldn't it be the third argument?

Reply:
@jbudz Options here is used for polluting the prototype that we're trying to guard against. 

At the point where `options.__proto__['2']` is assigned, we pollute the global object prototype.

Similar example:

```
const obj = {};
obj.__proto__['2'] = 'some command';

const arr = []
console.log(arr[2]) // logs "some command"

Reply:
Hey, thanks for helping me understand.  I'm still struggling a bit...

`command` is `const command = path.join('test', 'harden', '_echo.sh')`.  Running  `_echo.sh` with Node.js as the shell doesn't seem like it run anything.  

I toggled `require('../../src/setup_node_env');`.  The test failed with:
```
    operator: equal
    expected: ''
    actual:   'polluted!'
```
but it fails in the same way if I only run assertProcess().  How do I validate options has the potential for doing anything?